### DISCUSSION TITLE
Reuse Form elements for Frontend?

### DISCUSSION THREAD

**Discussion by wsdnathan** [orchidsoftware/platform#1563] [Q&A] [answered]
If I build a form with Orchid Form elements in the admin, how can I easily replicate that form on the frontend? Is there an example anywhere of reusing Form elements from Orchid, or should I be taking a different approach for the frontend? Basically I want to use all of this for a Contact form or RFP form https://orchid.software/en/docs/field/

**Comment by tabuna** [comment, marked as answer]
This package was not built to handle front-end tasks separately. But you can use fields like this:

https://github.com/orchidsoftware/platform/blob/c10871f4e61a96199a0f603d1ff7758c65b9c142/resources/views/auth/signin.blade.php#L7-L13
